src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -54,6 +54,57 @@
     short-summary: Manage certificates.
 """
 
+helps['keyvault certificate get-default-policy'] = """
+    type: command
+    short-summary: Get a default policy for a self-signed certificate
+    long-summary: >
+        This default policy can be used in conjunction with `az keyvault create` to create a self-signed certificate.
+        The policy can also be used as a base to manipulate to create other permutations of Key Vault certificates.
```

Comment:
> The policy can also be used as a base to manipulate to create other
        permutations of Key Vault certificates.

Does 'to manipulate to create' make sense?